sql/_project_record_ids.sql

```diff
@@ -27,26 +27,29 @@ dbscan AS (
 project_record_join AS (
 	SELECT 
 	    a.record_id,
-	    COUNT(record_id) OVER(PARTITION BY id) as records_in_project,
+	    COUNT(record_id) OVER(PARTITION BY id) AS records_in_project,
 	    a.id,
-	    (a.geom IS NULL)::integer as no_geom,
+	    (a.geom IS NULL)::integer AS no_geom,
 	    a.geom
 	FROM dbscan a
 ),
 all_intersections AS (
-	SELECT 
-		ST_UNION(ST_INTERSECTION(a.geom, b.geom)) as intersect_geom
+	SELECT
+		ST_UNION(ST_INTERSECTION(a.geom, b.geom)) AS intersect_geom
 	FROM  project_record_join a, project_record_join b
 	WHERE a.record_id < b.record_id
 	AND a.records_in_project > 1
 	AND b.records_in_project > 1
 	AND a.no_geom = 0
 	AND b.no_geom = 0
+	AND a.id = b.id
```

Comment:
This line is breaking the build - why are there two project_record_join tables (a,b)? I think that is the issue but don't fully understand the logic behind having an `a` and `b` of the same table